app/models/invenio_rdm_record_converter.rb

```diff
@@ -574,6 +577,21 @@ def original_identifiers(identifiers)
       }
     end
 
+    # if there are multiple isbn find the isbn with length 10 and remove it
+    if multiple_isbn
+      identifiers = normalize_isbn(identifiers)
+    end
+
     identifiers.compact
   end
+
+  def normalize_isbn(identifiers)
+    identifiers.map do |id_obj|
+      if id_obj.present? && id_obj[:scheme] == "isbn" && id_obj[:identifier].to_s.length == 10
+        nil
+      else
+        id_obj
+      end
+    end
```

Comment:
The issue here is that if the record only has 1 ISBN10, it will be removed. `multiple_isbn` is set to true when there is a single isbn instead of when there are multiple. It's fair to assume there is at most only 1 ISBN of each type per record though.